packages/cli/src/commands/generate/function/function.js

```diff
@@ -54,13 +53,44 @@ export const builder = (yargs) => {
         'https://redwoodjs.com/reference/command-line-interface#generate-function'
       )}`
     )
+
   // Add default options, includes '--typescript', '--javascript', '--force', ...
   Object.entries(yargsDefaults).forEach(([option, config]) => {
     yargs.option(option, config)
   })
 }
 
-export const { command, handler } = createYargsForComponentGeneration({
-  componentName: 'function',
-  filesFn: files,
-})
+export const handler = async ({ force = false }) => {
```

Comment:
Is this correct to create my own handler - so I can console log some messaging after the tasks run?

Reply:
Looks like it is, I just was not setting the command and some params approriately.

![image](https://user-images.githubusercontent.com/1051633/113510677-f74abc00-9529-11eb-978d-c24c21053a2c.png)